templates/Collector/db.html.twig

```diff
@@ -384,6 +393,22 @@
             </div>
         </div>
 
+        <div class="tab">
+            <h3 class="tab-title">Managed Entities</h3>
+            <div class="tab-content">
+                {% if not collector.managedEntityCountByClass %}
+                    <div class="empty">
+                        <p>No mapped entities.</p>
+                    </div>
+                {% else %}
+                    {% for manager, entityCounts in collector.managedEntityCountByClass %}
+                        <h3>{{ manager }} <small>entity manager</small></h3>
```

Comment:
should be a `<h4>` as the tab title is already a `h3`

Reply:
I've updated it but the entity mapping tables are also `h3`s. Should I also update them to be h4?

Reply:
I would say yes

Reply:
Done